src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
```

Comment:
:information_source: Avoid using @author tag as it is not recommended in modern Java development. (Severity: Minor)